### DISCUSSION TITLE
editor.auto-save.focus-lost should save when using ctrl+z to put helix in the background

### DISCUSSION THREAD

**Discussion by robinmoussu** [helix-editor/helix#11821] [Ideas] [answered]
I was trying to set-up file autosave, and discovered when reading #2542 that you can save file automatically when helix [lose focus](https://docs.helix-editor.com/editor.html#editorauto-save-section). I would have expected that using ctrl+z (to put helix in the background) would be considered as a "focus lost" event, and thus my files would be saved.

**Comment by the-mikedavis** [comment, marked as answer]
The focus lost event is about the terminal emulator window or pane rather than the application within. You should be able to rebind `C-z` to save and then suspend though:

```toml
[keys.normal]
C-z = [":write-all", "suspend"]
```
